documentation/explanation/architecture.md

```diff
@@ -0,0 +1,13 @@
+# Charmed Superset Architecture
```

Comment:
The product architecture was not part of the [Documentation objectives commitments](https://docs.google.com/spreadsheets/d/1wrsRvUdS7PB9v2Huwgei8p7ymoGcxBXQVDZGbOZbh6g/edit?gid=1661754936#gid=1661754936) for this cycle, and therefore I would keep this document still as work in progress and not published on Charmhub. I suggest so because a reference architecture should provide more information about each component, how they interact with each other and potential interactions with the rest of the Canonical stack, i.e., in general I am missing more information. You can find an attempt to do this [here](https://documentation.ubuntu.com/data-science-stack/en/latest/explanation/dss-arch/). We can discuss further in a call, for now I would keep it just for internal use.

Reply:
Btw, nice diagram!

Reply:
Ok sounds good, I'll leave this in the repo but not publish it to charmhub :)